Resources/Prototypes/Entities/Mobs/NPCs/miscellaneous.yml

```diff
@@ -79,6 +79,57 @@
   name: tomato killer
   description: Looks like it's not you eating tomatoes today, it's the tomatoes eating you.
   components:
+  - type: GhostRole
+    prob: 0.15
+    makeSentient: true
+    allowSpeech: true
+    name: ghost-role-information-tomatokiller-name
+    description: ghost-role-information-tomatokiller-description
+    rules: ghost-role-information-familiar-rules
+    raffle:
+      settings: short
+  - type: GhostTakeoverAvailable
+  - type: CanEscapeInventory
+    baseResistTime: 3
+  - type: Puller
+    needsHands: false
+  - type: Reactive
+    groups:
+      Flammable: [ Touch ]
+      Extinguish: [ Touch ]
+    reactions:
+    - reagents: [ Water, SpaceCleaner ]
+      methods: [ Touch ]
+      effects:
+      - !type:WashCreamPieReaction
```

Comment:
Is this part needed? As far as I know the tomatos are immune to banana cream pies.

Reply:
Perhaps, but the component does not work for me without this part. Perhaps there is a way to do without it

Reply:
I just gave it a test and it works fine. Remove lines 101 to 104.